.circleci/config.yml

```diff
@@ -164,6 +164,53 @@ jobs:
       - run: make -C build coverage
       - run: /bin/bash <(curl -s https://codecov.io/bash) || echo "Codecov did not collect coverage reports"
 
+  clang-build-debug:
+    docker:
+      - image: valhalla/valhalla:build-latest
+    steps:
+      - checkout
+      - run: ./scripts/format.sh && ./scripts/error_on_dirty.sh
+      - run: node ./scripts/validate_changelog.js
+      - run: git submodule sync && git submodule update --init
+      - restore_cache:
+          keys:
+            - ccache-debug-linux-x86_64-{{ .Branch }}
+            - ccache-debug-linux-x86_64
+      - run: npm install --ignore-scripts
+      - run: mkdir build
+      - run: ls -la /usr/bin/
+      - run: clang-5.0 --version
```

Comment:
@SvetlanaBayarovich what version of Clang are you running? Why [Clang 5](https://releases.llvm.org/5.0.0/tools/clang/docs/ReleaseNotes.html) here vs something more recent? Is it the version of Clang supported by the CircleCI image for this workflow? I would be in-favor of turning the `build-lint-debug` CI task into one using [clang 10](https://releases.llvm.org/10.0.0/tools/clang/docs/index.html).

What I'm currently running: 

```
$ clang --version
clang version 10.0.0
Target: x86_64-apple-darwin19.5.0
Thread model: posix
InstalledDir: /usr/local/opt/llvm/bin
```

Reply:
The story behind this is the following. I was trying to understand how to make CI with address sanitizer. The problem was that it failed with the error 
```
[ 97%] Generating node_valhalla.log
==23592==ASan runtime does not come first in initial library list; you should either link runtime to your application or manually preload it with LD_PRELOAD.
npm ERR! Test failed.  See above for more details.
test/CMakeFiles/run-node_valhalla.dir/build.make:70: recipe for target 'test/node_valhalla.log' failed
```
My initial idea was that it was happening due to the lack of support on the side of `gcc`, that's why I was trying `clang`. As I found out by searching `build-base` logs, the clang version that is being installed is 5.0. However it looks like the compiler is not the reason of the issue.